packages/dd-trace/test/appsec/rasp/utils.spec.js

```diff
@@ -152,6 +152,52 @@ describe('RASP - utils.js', () => {
       expect(abortError.blockingAction).to.equal(result.actions.blocking_action)
     })
 
+    it('should create DatadogRaspAbortError when blockingAction is present in fastify', () => {
+      const rootSpan = {
+        context: sinon.stub().returns({ _name: 'fastify.request' })
+      }
+      const abortController = {
+        abort: sinon.stub(),
+        signal: {}
+      }
+      const result = {
+        actions: {
+          blocking_action: { type: 'block_request' }
+        }
+      }
+
+      web.root.returns(rootSpan)
+
+      utils.handleResult(result, req, res, abortController, config, raspRule)
+
+      sinon.assert.calledOnce(abortController.abort)
+      const abortError = abortController.abort.firstCall.args[0]
+      expect(abortError).to.be.instanceOf(utils.DatadogRaspAbortError)
+      expect(abortError.raspRule).to.equal(raspRule)
+      expect(abortError.blockingAction).to.equal(result.actions.blocking_action)
+    })
```

Comment:
Since the only difference between the two tests is the root span name (`express.request` vs `fastify.request`), we could parametrize the test with a loop. Maybe something like this:

```js
['express', 'fastify'].forEach(supportedWebFramework => {
    it(`should create DatadogRaspAbortError when blockingAction is present in ${supportedWebFramework}`, () => {
        const rootSpan = {
            context: sinon.stub().returns({ _name: `${supportedWebFramework}.request` })
        }
        
        {...}
    }
})
```

That way we avoid copy-pasting the exact same logic and make it easier to extend later.